starter-code/lib/office.js

```diff
@@ -0,0 +1,28 @@
+var opinions = ["This is the best job ever",
```

Comment:
Buen trabajo. Se nota el esfuerzo. Seguid así estáis en ascenso. Intentad este fin de semana pegarle una vuelta, sin _. y así trabajáis con Arrays.